understanding/21/orientation.html

```diff
@@ -31,11 +31,11 @@ <h3>Benefits</h3>
 			<h2>Examples</h2>
 			<ul>
 				<li><strong>Example 1: Online video site</strong><br />
-				Videos are not locked to a specific orientation.  A user can start a video with their device in a portrait orientation, resulting in a smaller video being displayed and then rotate the device to landscape and have the video expand to fill the entire screen.  All controls and functionality are available in either orientation.</li>
+				Videos are not locked to a specific orientation.  A user can start a video with their device in portrait or in landscape orientation.</li>
```

Comment:
Comment from MichaelG: "A video is shown in either portrait or in landscape based on the user's chosen orientation."